modules/react/src/components/export-video/export-video-modal.js

```diff
@@ -70,7 +70,7 @@ class ExportVideoModal extends Component {
       content: {
         top: 'auto',
         left: 'auto',
-        right: `calc(50% - ${DIMENSIONS.sidePanel.width / 2}px)`,
+        right: `calc(50% - ${SIDEPANEL_WIDTH / 2}px)`,
```

Comment:
Ah, I see how this is necessary to import. If kepler's styled-components theme provides this we could pull it from there, but I suppose even that is unnecessary for this usage.

Reply:
Yeah I think `export-video-modal.js` is mainly useful for demonstration. It's not used with a custom modal rendering ExportVideoContainer directly.